Source/WebKit/UIProcess/API/Cocoa/WKWebsiteDataStore.mm

```diff
@@ -299,6 +299,35 @@ + (void)_deleteDefaultDataStoreForTesting
     return WebKit::WebsiteDataStore::deleteDefaultDataStoreForTesting();
 }
 
++ (void)_fetchAllIdentifiers:(void(^)(NSArray<NSUUID *> *))completionHandler
+{
+    auto completionHandlerCopy = makeBlockPtr(completionHandler);
+    WebKit::WebsiteDataStore::fetchAllDataStoreIdentifiers([completionHandlerCopy](auto&& identifiers) {
+        auto result = adoptNS([[NSMutableArray alloc] init]);
+        for (auto identifier : identifiers) {
+            auto uuid = adoptNS([[NSUUID alloc] initWithUUIDString:identifier.toString()]);
```

Comment:
initWithUUIDBytes might skip the String roundtrip.
Also, WTF::UUID could be given a constructor that takes an NSUUID and a getter that returns a RetainPtr<NSUUID> to avoid duplicate code like this.

Reply:
Found it in CocoaHelpers.h.

Reply:
hmm I guess we are not going to use the ones in CocoaHelpers.h due to endianness; I added functions to UUID.h